pillar/base/bugs.sls

```diff
@@ -32,7 +32,7 @@ bugs:
             spambayes_uri: "http://localhost:8001/sbrpc"
             spambayes_ham_cutoff: "0.2"
             spambayes_spam_cutoff: "0.85"
-            ciavc_server: "http://CIA.vc"
+            ciavc_server: "https://python.org"
```

Comment:
My only reason for now just outright removing this is I am unsure if it will cause any breakage/outage or something not to load and spam logs.

Reply:
bet it is safe:

```
ee@bugs:/srv/roundup$ sudo grep -r 'ciavc_server' /srv/roundup/src
ee@bugs:/srv/roundup$ sudo grep -r 'ciavc' /srv/roundup/src
ee@bugs:/srv/roundup$ 
```

Reply:
Good deal! Removing then